src/test/ui/regions/forall-wf-ref-reflexive.rs

```diff
@@ -0,0 +1,18 @@
+// Test that we consider `for<'a> &'a T: 'a` to be sufficient to prove
+// that `for<'a> &'a T: 'a`.
+//
+// FIXME. Except we don't!
```

Comment:
known-bug?

Reply:
I believe so, certainly a pre-existing one. This feels like an implied bounds situation to me. We ought to be able to know that `for<'a> &'a T: 'a` is really `for<'a> if (WF(&'a T)) { &'a T: 'a }`, and -- in that case -- we would be succesful here.

Reply:
I think @jackh726 means using the `// known-bug` attribute

Reply:
Ah, cool. Citing which bug?